docs/source/02_installation.md

```diff
@@ -1,4 +1,4 @@
-# Building ADAM from Source {#build-from-source}
+# Building ADAM from Source
```

Comment:
ok, if you log into readthedocs, you should be able to go to https://readthedocs.org/dashboard/adam and hit the Admin button